patch-capfram.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+cd node_modules/capital-framework/src/cf-core/src
+sed -i '' -e 's|../../|~|g' cf-core.less
```

Comment:
@jimmynotjim Those were the only two that threw warnings for this set of components.  I am sure there are more